src/bsmHandler.cpp

```diff
@@ -36,7 +36,7 @@
 #include "cvlib.hpp"
 #include "bsmHandler.hpp"
 #include "spdlog/spdlog.h"
-#include "../../include/redaction-properties/RedactionPropertiesManager.hpp"
+#include "../../include/general-redaction/redactionPropertiesManager.hpp"
```

Comment:
It seems the CMakeList.txt has already included these header files. Can we include the header file with name redactionPropertiesManager.hpp here? The same comments for other headers file that are included using relative path.

Reply:
Relative paths have been removed and we're now referencing our header files directly in the following files:
- src/bsm.cpp
- src/general-redaction/rapidjsonRedactor.cpp
- src/general-redaction/redactionPropertiesManager.cpp
- src/idRedactor.cpp
- src/kafka_consumer.cpp
- src/ppm.cpp
- src/ppmLogger.cpp
- src/tests.cpp
- src/velocityFilter.cpp

Relevant CDOT PR: https://github.com/CDOT-CV/jpo-cvdp/pull/18